tests/functional/user-data-manager.test.js

```diff
@@ -51,7 +52,42 @@ describe('indexeddb-workflow.test.js', () => {
     jest.clearAllMocks()
   })
 
-  it('1 IndexedDBWorkflow - delete many and update method, checking blocking', async () => {
+  async function prepareWordItem (word, lang = Constants.LANG_LATIN) {
+    let langCode = LMF.getLanguageCodeFromId(lang)
+    let adapterTuftsRes = await ClientAdapters.morphology.tufts({
+      method: 'getHomonym',
+      params: {
+        languageID: lang,
+        word: word
+      }
+    })
+
+    let testHomonym = adapterTuftsRes.result
+    
+    let context = []
+    let tqselector = TextQuoteSelector.readObject({
+      languageCode: langCode,
+      targetWord: word,
+      target: {
+        source: 'foosource',
+        selector: {
+          exact: word,
+          prefix: 'fooprefix',
+          suffix: 'foosuffix'
+        }
+      }
+    })
+    context.push(tqselector)
+
+    return new WordItem({
+      targetWord: word,
+      languageCode: langCode,
+      homonym: testHomonym,
+      context: context
+    })
+  }
+
+  it('1 UserDataManager - delete many and update method, checking blocking', async () => {
```

Comment:
does this actually test the blocking condition? It would only do so if a later request is started while an earlier is still processing, correct? If this code does actually test that condition, can you add some comments to explain how? thanks!

Reply:
Of course, here are some explanations:
1) we create Promises (without await) for 4 updates and 1 deleteMany
2) After we execute them all together here
```
let final = [await res1, await res2, await res3, await res4, await res5]
```
It works similiar to Promise.all
(https://stackoverflow.com/questions/35612428/call-async-await-functions-in-parallel#answer-45772148)

Reply:
ok, great, thank you :-)